Core/src/org/sleuthkit/autopsy/modules/encryptiondetection/EncryptionDetectionFileIngestModule.java

```diff
@@ -134,17 +136,39 @@ public void startUp(IngestJobContext context) throws IngestModule.IngestModuleEx
                  */
                 if (!file.getKnown().equals(TskData.FileKnown.KNOWN)) {
                     /*
-                     * Qualify the MIME type.
+                     * Has the file been deleted?
                      */
-                    String mimeType = fileTypeDetector.getMIMEType(file);
-                    if (mimeType.equals("application/octet-stream")) {
-                        if (isFileEncryptionSuspected(file)) {
-                            return flagFile(file, BlackboardArtifact.ARTIFACT_TYPE.TSK_ENCRYPTION_SUSPECTED,
-                                    String.format(Bundle.EncryptionDetectionFileIngestModule_artifactComment_suspected(), calculatedEntropy));
+                    if (!file.isMetaFlagSet(TskData.TSK_FS_META_FLAG_ENUM.UNALLOC)
+                            && !file.isDirNameFlagSet(TskData.TSK_FS_NAME_FLAG_ENUM.UNALLOC)) {
+                        /*
+                         * Is the file in FILE_IGNORE_LIST?
+                         */
+                        boolean ignoreFile = false;
```

Comment:
Personally, I find code that sets flags and then checks them in later conditionals can quickly become hard to read and maintain. I'm thinking we can just return immediately if we find one of the files we are ignoring.